light_gallery/templates/light_gallery.html

```diff
@@ -11,14 +11,38 @@
 {% addtoblock "css" %}
 <link type="text/css" rel="stylesheet" href="{% static 'css/django-light-gallery.min.css' %}" />
 <style>
-    .djangocms-light-gallery__link {
+    .djangocms-light-gallery__outer-container {
         -ms-flex-preferred-size: calc(100% / {{ thumbnails_per_row }});
         flex-basis: calc(100% / {{ thumbnails_per_row }});
     }
 
+    {% if thumbnail_border_size %}
+        .djangocms-light-gallery__image-container {
+            border-width: {{ thumbnail_border_size }}px;
+            border-style: solid;
+            border-color: {% firstof thumbnail_border_color 'black' %};
+        }
+
+        .djangocms-light-gallery__image--horizontal {
+            margin-right: {{ thumbnail_padding_size }}px;
```

Comment:
there is no guard for `thumbnail_padding_size` here (doesn't appear to affect anything presentation wise; I suppose the rule is ignored?).

Reply:
(ditto below)